static/js/stream_list.js

```diff
@@ -269,6 +274,7 @@ exports.create_sidebar_row = function (sub) {
         return;
     }
     build_stream_sidebar_row(sub);
+    $('.stream-name[data-toggle="tooltip"]').tooltip();
```

Comment:
Is there a way to make the selector apply only to the row you just build, maybe by moving this call?  in `build_stream_sidebar_row` you have access to `list_item`.